libs/blocks/card/cardUtils.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/prefer-default-export */
+import { createTag } from '../../utils/utils.js';
+import { getMetadata } from '../section-metadata/section-metadata.js';
+
+const DOUBLE_WIDE = 'DoubleWideCard';
+
+export const addBackgroundImg = (picture, cardType, card) => {
+  const url = picture.querySelector('img').src;
+  const imageDiv = document.createElement('div');
+
+  imageDiv.style.backgroundImage = `url(${url})`;
+  imageDiv.classList.add(`consonant-${cardType}-img`);
+  card.append(imageDiv);
+};
+
+const getUpFromSectionMetadata = (section) => {
+  const sectionMetadata = section.querySelector('.section-metadata');
+  if (!sectionMetadata) return null;
+  const metadata = getMetadata(sectionMetadata);
+  const styles = metadata.style?.text.split(', ').map((style) => style.replaceAll(' ', '-'));
+  return styles?.find((style) => style.includes('-up'));
+};
+
+export const addFooter = (links, container, merch) => {
+  const linksArr = Array.from(links);
+  const linksLeng = linksArr.length;
+  const hrTag = merch ? '<hr>' : '';
+  let footer = `<div class="consonant-CardFooter">${hrTag}<div class="consonant-CardFooter-row" data-cells="1">`;
+  footer = linksArr.reduce(
+    (combined, link, index) => (
+      `${combined}<div class="consonant-CardFooter-cell consonant-CardFooter-cell--${(linksLeng === 2 && index === 0) ? 'left' : 'right'}">${link.outerHTML}</div>`),
+    footer,
+  );
+  footer += '</div></div>';
+
+  container.insertAdjacentHTML('beforeend', footer);
+  links.forEach((link) => {
+    if (link.parentElement && document.body.contains(link.parentElement)) {
+      link.parentElement.remove();
```

Comment:
you don't need curly brackets for a single line

Reply:
@VKniaz - this is still open

Reply:
removed